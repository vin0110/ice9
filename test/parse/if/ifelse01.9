if true -> a; [] else -> b; fi
