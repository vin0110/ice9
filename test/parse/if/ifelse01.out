IF:
  BLIT: TRUE
(then)
  ID:'a'
(else)
  ID:'b'
